setup.py

```diff
@@ -36,5 +41,6 @@
     py_modules=[module.stem for module in here.glob("src/*.py")],
     package_dir={"": "src"},
     include_package_data=True,
-    python_requires=">=3.5",
+    install_requires=['typing;python_version<"3.5"'],
```

Comment:
`typing` should not be required for installation of this package.
Please delete this configuration.